config/ember-try.js

```diff
@@ -1,20 +1,28 @@
-/*jshint node:true*/
+/* eslint-env node */
 module.exports = {
+  useYarn: true,
   scenarios: [
     {
-      name: 'default',
-      bower: {
-        dependencies: { }
-      }
-    },
-    {
-      name: 'ember-1.13',
+      name: 'ember-lts-2.8',
       bower: {
         dependencies: {
-          'ember': '~1.13.0'
+          'ember': 'components/ember#lts-2-8'
         },
         resolutions: {
-          'ember': '~1.13.0'
+          'ember': 'lts-2-8'
+        }
+      },
+      npm: {
+        devDependencies: {
+          'ember-source': null
+        }
+      }
+    },
+    {
+      name: 'ember-lts-2.12',
+      npm: {
+        devDependencies: {
+          'ember-source': '~2.12.0'
```

Comment:
Curious, what's the difference between `components/ember#lts-2-8` and just specifying the version number?

Reply:
I think the `lts` tag isn't necessarily applied to `2.8.0` - the `lts` will pick up whichever 2.8.x release got the `lts` designation. (I just went with `ember-cli`'s default `ember-try` blueprint here.)

Reply:
Sounds good to me! :D